core/mapper/exprmapper/link/link.go

```diff
@@ -0,0 +1,47 @@
+package link
```

Comment:
link is part of a flow and should be in flogo-contrib... @fm-tibco ?

Reply:
Yes. I will remove it from flogo-lib